### PR TITLE
RTCP packets table

### PR DESCRIPTION
**Pull request by LVala**

### CODE REVIEW

File: client/src/app/gui/rtcp_packets_table.rs [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+        let mut last_id = 0;
+        let mut next_ix = 1;
+
+        let first_ts = streams.packets.get(0).unwrap().timestamp;
+        body.heterogeneous_rows(heights, |ix, mut row| {
+            let (id, rtcp) = rtcp_packets.get(ix).unwrap();
+            let packet = streams.packets.get(*id).unwrap();
+
+            if *id != last_id {
+                log::info!("DUPA {}, {}", *id, last_id);
```
Comment (Falon452) [inline client/src/app/gui/rtcp_packets_table.rs:86]: Can be removed

File: client/src/app/gui/rtcp_packets_table.rs [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+                ui.label(format!("{:.4}", timestamp.as_secs_f64()));
+            });
+            row.col(|ui| {
+                ui.label(packet.source_addr.to_string());
+            });
+            row.col(|ui| {
+                ui.label(packet.destination_addr.to_string());
+            });
+            row.col(|ui| {
+                ui.label(rtcp.get_type_name().to_string());
```
Comment (Falon452) [inline client/src/app/gui/rtcp_packets_table.rs:106]: ui.label(RichText::from(rtcp.get_type_name()).strong());

Comment (Falon452) [inline client/src/app/gui/rtcp_packets_table.rs:106]:
```suggestion
                ui.label(RichText::from(rtcp.get_type_name()).strong());
```

**Review by Falon452 — APPROVED** [review]

File: client/src/app/gui/rtcp_packets_table.rs (hunk 1 above)
Comment (LVala) [reply client/src/app/gui/rtcp_packets_table.rs:86]: 😳

File: client/src/app/gui/rtcp_packets_table.rs (hunk 2 above)
Comment (LVala) [reply client/src/app/gui/rtcp_packets_table.rs:106]: Tried it and honestly, I don't think it looks better. Gonna leave as it is, we can always change it later.

**Pull request merged**